src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp

```diff
@@ -105,85 +103,62 @@ inline fs::path czechPkcs11ModulePath()
 #endif
 }
 
-const std::map<Pkcs11ElectronicIDType, Pkcs11ElectronicIDModule> SUPPORTED_PKCS11_MODULES {
+const std::map<ElectronicID::Type, Pkcs11ElectronicIDModule> SUPPORTED_PKCS11_MODULES {
     // EstEIDIDEMIAV1 configuration is here only for testing,
```

Comment:
Comment is now out of sync as it still mentions `EstEIDIDEMIAV1`

Reply:
Thank you